src/test/cypress/cypress/integration/UserCard.spec.js

```diff
@@ -1246,11 +1245,11 @@ describe('User Card ', function () {
       usercard.preview();
 
       // We check Control Center FR East is displayed in 'Recipients' section and not in 'Recipients for information'
-      cy.get("#opfab-entity-recipients").find('span').should('have.length', 3);
+      cy.get("#opfab-entity-recipients").find('span').should('have.length', 5);
       usercard.checkEntityRecipientsInPreviewContains("Control Center FR North");
       usercard.checkEntityRecipientsInPreviewContains("Control Center FR East");
 
-      cy.get("#opfab-entity-recipients-for-information").find('span').should('have.length', 2);
+      cy.get("#opfab-entity-recipients-for-information").find('span').should('have.length', 3);
```

Comment:
Checking for the number of spans in this section is not relevant (lines 1248 and 1252)